lib/services/batch/lib/models/autoPoolSpecification.js

```diff
@@ -56,15 +58,9 @@ class AutoPoolSpecification {
    * 5 - OS Family 5, equivalent to Windows Server 2016. For more information,
    * see Azure Guest OS Releases
    * (https://azure.microsoft.com/documentation/articles/cloud-services-guestos-update-matrix/#releases).
-   * @member {string} [pool.cloudServiceConfiguration.targetOSVersion] The
-   * default value is * which specifies the latest operating system version for
-   * the specified OS family.
-   * @member {string} [pool.cloudServiceConfiguration.currentOSVersion] This
-   * may differ from targetOSVersion if the pool state is Upgrading. In this
-   * case some virtual machines may be on the targetOSVersion and some may be
-   * on the currentOSVersion during the upgrade process. Once all virtual
-   * machines have upgraded, currentOSVersion is updated to be the same as
-   * targetOSVersion.
+   * @member {string} [pool.cloudServiceConfiguration.osVersion] The default
```

Comment:
These `@member` annotations should have all been changed over to `@property`. Can you please run `autorest --reset` and then try regenerating?